apps/tlon-mobile/cosmos.imports.ts

```diff
@@ -16,7 +16,7 @@ import * as fixture1 from './src/fixtures/TlonButton.fixture';
 import * as decorator0 from './src/fixtures/cosmos.decorator';
 
 export const rendererConfig: RendererConfig = {
-  playgroundUrl: 'http://localhost:5000',
+  playgroundUrl: 'http://localhost:5001',
```

Comment:
Intentional?

Reply:
Yeah, 5000 is used by the system on Mac if you have AirPlay Receiver enabled ([see here](https://github.com/react-cosmos/react-cosmos/issues/1355))

Fun fact: that feature works well for mirroring your phone screen to your laptop for demo's/Gather screen share